challenges/classes.js

```diff
@@ -1,11 +1,62 @@
 // Copy and paste your prototype in here and refactor into class syntax.
 
+/*  function CuboidMaker (l, w, h) {
+    this.l = l
+    this.w = w
+    this.h = h
+}
+
+CuboidMaker.prototype.volume = function(l, w, h) {
+  return this.l * this.w * this.h
+}
+
+CuboidMaker.prototype.surfaceArea = function(l, w, h) {
+  return 2 * (this.l * (this.w + this.h) + (this.w * this.h))
+}
+       */
+
+class CuboidMaker {
+  constructor(l, w, h){
```

Comment:
Only one param is needed. `dimensions` then use dot notation to access its properties.